packages/components/src/JSONSchemaRenderer/JSONSchemaRenderer.component.js

```diff
@@ -72,6 +72,20 @@ function TextRenderer({ propertyKey, title, properties }) {
 	);
 }
 
+function PasswordRenderer({ propertyKey, title, properties }) {
```

Comment:
please export and test

Reply:
tested through JSONSchemaRenderer on line 133